packages/manager/apps/carrier-sip/src/routing.js

```diff
@@ -12,6 +12,20 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => CarrierSipService.getCarrierSip(billingAccount, serviceName),
       cdrsLink: /* @ngInject */ ($state, billingAccount, serviceName) =>
         $state.href('app.cdrs', { billingAccount, serviceName }),
+      clusterDetails: /* @ngInject */ (
+        $http,
+        $q,
+        billingAccount,
+        serviceName,
+      ) =>
+        $http
+          .get(
+            `/telephony/${billingAccount}/carrierSip/${serviceName}/clusterDetails`,
+          )
+          .then(({ data }) => data)
+          .catch((error) =>
+            [404, 500].includes(error.status) ? {} : $q.reject(error),
```

Comment:
500 is legit ?

Reply:
Good question, it depends if we consider that this specific routes can crash independently from top carrierSip route.

Reply:
@JeremyDec If the route can crash independently then all errors should be catched not only 404 and 500

Reply:
@marie-j that was done at first (see @frenautvh first comment), so I had to choose.